.circleci/config.yml

```diff
@@ -266,6 +317,17 @@ workflows:
           filters:
             branches:
               only: master
+      - publish-image:
+          name: publish-service-builder-20
```

Comment:
For consistent ordering, I think this should go below the `publish-20-run-stack` step

Reply:
+1